src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-style: book;
```

Comment:
Consider using a more descriptive font-style value like 'normal' instead of 'book'.